### PR TITLE
Palantir/RetailMeNot Sizing

### PR DESCRIPTION
**Pull request by redsn0w422**
- Fixed RetailMeNot and Palantir logos on small screens
- Fixed speaker card sizing on mobile

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -544,23 +550,35 @@ ul > li > a {
...
+		width: calc(50% - 5%);*/
 	}
 	.gigabyte-sponsors img {
 		max-width: 200px;
 		max-height: 200px;
+		height: 10%;
+		/*width: 100%;
+		margin-top: 0;
+		margin-bottom: 0;
+		width: calc(50%*/ - 5%);
```
Comment (sskhandek) [inline assets/css/styles.css]: @redsn0w422 what's going on here?

**Comment by sskhandek** [comment]
@redsn0w422 Try to modularize you PRs a bit. The work you did on the speaker cards should be in a separate PR so that we can do a fix on the logos first.

@bluedrops can I get your advice on the speaker card layout? you were doing work on that earlier.

**Comment by redsn0w422** [comment]
Okay I'll move the speaker card stuff to a new PR. 

Also oops that was me playing with the spacing, I'll take that out.

**Comment by sskhandek** [comment]
Thanks @redsn0w422

**Comment by bluedrops** [comment]
@sskhandek Sorry for missing this; what did you need advice on regarding the speaker card layout?

**Pull request merged**